pkgs/development/tools/mold/default.nix

```diff
@@ -33,6 +33,10 @@ stdenv.mkDerivation rec {
     mimalloc
   ];
 
+  patches = [
+    ./fix-debug-strip.patch # fix --debug-strip; https://github.com/rui314/mold/pull/1038
```

Comment:
Isn't this usually done through `fetchpatch` when they are available on GitHub?

Reply:
I did it this way specifically because the patch is actually in a fork, not the main repo or branch, and GitHub can and will garbage collect things eventually if the branch was merged and deleted, I think. Honestly the cases where it will GC is sometimes confusing, IME. (I have no clue when the next mold release will happen *or* when that patch will get merged, so it's just a precaution.)

Reply:
Just to be clear, we can and should migrate this to `fetchpatch` if I'm wrong, or at least once the patch is merged into main, and delete the local file. At worst case I guess if we did it now, all that could happen is the build breaking, though that would be a shame.